rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -280,50 +280,28 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> xSettings, const
 	auto l_OnSearchBoxTextChanged = [=](QString text)
 	{
 		QString searchTerm = text.toLower();
-		QList<QListWidgetItem*> checked_Libs;
-		QList<QListWidgetItem*> unchecked_Libs;
+		QList<QListWidgetItem*> list;
 
-		// create sublists. we need clones to preserve checkstates
-		for (int i = 0; i < ui->lleList->count(); ++i)
+		// create list. we need clones to preserve checkstates
+		for (int i = 0; i < ui->lleList->count(); i++)
 		{
-			if (ui->lleList->item(i)->checkState() == Qt::Checked)
-			{
-				checked_Libs.append(ui->lleList->item(i)->clone());
-			}
-			else
-			{
-				unchecked_Libs.append(ui->lleList->item(i)->clone());
-			}
+			list.append(ui->lleList->item(i)->clone());
 		}
 
-		// sort sublists
-		auto qLessThan = [](QListWidgetItem *i1, QListWidgetItem *i2) { return i1->text() < i2->text(); };
-		qSort(checked_Libs.begin(), checked_Libs.end(), qLessThan);
-		qSort(unchecked_Libs.begin(), unchecked_Libs.end(), qLessThan);
+		// sort list
+		qSort(list.begin(), list.end(), [](QListWidgetItem *i1, QListWidgetItem *i2) {
```

Comment:
Please DON'T use `qSort()`. `QAlgorithms` have been depreacted for quite some time. Just use the std ones (i.e. in this case just a regular `std::sort`).